Dockerfile

```diff
@@ -132,7 +132,7 @@ RUN go get github.com/campoy/embedmd@97c13d6 \
 
 # Install liche (Markdown link checker) (via Go).
 # https://github.com/raviqqe/liche
-RUN go get github.com/raviqqe/liche@3ac05a3 \
+RUN go get github.com/raviqqe/liche@f9ba5f2 \
```

Comment:
Could you please add a Date or Version comment, so it's easier to understand what to update later?

Reply:
Unfortunately there's no version, so the update here is just to get the latest commit at head. Even a comment with the date does not really help I think, does it?

Reply:
I often put a date just to understand how old the version is.